src/java/com/zimbra/qa/unittest/TestBulkProvision.java

```diff
@@ -223,7 +223,7 @@ public static String generateProvisioningXML() throws Exception {
                     //ignore
                 }
             }
-            Closeables.closeQuietly(stringWriter);
+            IOUtils.closeQuietly(stringWriter);
```

Comment:
Deprecation statement for Cloesables.closeQuietly:
Deprecated:
    Where possible, use the try-with-resources statement if using JDK7 or Closer on JDK6 to close one or more Closeable objects. This method is deprecated because it is easy to misuse and may swallow IO exceptions that really should be thrown and handled. See Guava issue 1118 for a more detailed explanation of the reasons for deprecation and see Closing Resources for more information on the problems with closing Closeable objects and some of the preferred solutions for handling it correctly. This method is scheduled to be removed in Guava 16.0.

Why not use "try with resources"?  Then you can remove the new dependency on IOUtils too.